ui/lib/kv/addon/components/page/secret/details.js

```diff
@@ -35,6 +35,7 @@ export default class KvSecretDetails extends Component {
 
   @tracked showJsonView = false;
   @tracked wrappedData = null;
+  @tracked syncStatus = null;
```

Comment:
All of sudden the test that was added to check for the sync status banner was failing and I noticed that this wasn't added as a tracked property. Not sure how it was passing before. After merging main there was something else added to the constructor which must have changed the timing of things.

Reply:
It was! It looks like it got removed when main was merged
https://github.com/hashicorp/vault/pull/24208/files#diff-2a7dbc4c7b51c6a2de3641f47e1598b8c2c7aaf59d442db2afbb5f4c36f54ba9

Reply:
previously there was a comment with it, not sure if you want to add it back or not:
```js
@tracked syncStatus; // array of association sync status info by destination